docs/data-apps/setup/index.md

```diff
@@ -0,0 +1,173 @@
+GRANT MONITOR ON ALL SCHEMAS IN database <database_with_snowplow_data> TO ROLE DATA_APPS_ROLE;
+GRANT SELECT ON ALL TABLES IN database <database_with_snowplow_data> TO ROLE DATA_APPS_ROLE;
+GRANT SELECT ON ALL VIEWS IN database <database_with_snowplow_data> TO ROLE DATA_APPS_ROLE;
+
+CREATE USER IF NOT EXISTS DATA_APPS_USER
+  PASSWORD='<user_password>'
+  MUST_CHANGE_PASSWORD = FALSE
+  DEFAULT_ROLE = DATA_APPS_ROLE
+  EMAIL = '<user_email>';
+GRANT ROLE DATA_APPS_ROLE TO USER DATA_APPS_USER;
+```
+
+Once created share the following details back with us:
+
+1. Snowflake Region
+2. Snowflake Account (we need the account identifier which you can find following this guide: https://docs.snowflake.com/en/user-guide/admin-account-identifier#finding-the-organization-and-account-name-for-an-account)
+3. Snowflake Username
+4. Snowflake Password (**please send this in a secure format)**
+5. Snowflake Database
+6. Snowflake Schema (indicator of where the app should default into but is configurable in most apps once launched)
+7. Snowflake Warehouse
+8. Snowflake Role
+
+</TabItem>
+<TabItem value="databricks" label="Databricks">
+
+Create a new [Service Principal](https://docs.databricks.com/en/administration-guide/users-groups/service-principals.html#add-service-principals-to-your-account-using-the-account-console) in your workspace, make sure to give it Databricks SQL access and Workspace access in the configuration. Alternatively create a [new user](https://docs.databricks.com/en/administration-guide/users-groups/users.html#add-users-to-your-account-using-the-account-console) and ensure they have those accesses.
+
+![Databricks account setup](./images/databricks-acc-setup.png)
+
+```sql
+
+// Uses the databricks inheritance model: https://docs.databricks.com/en/data-governance/unity-catalog/manage-privileges/privileges.html#inheritance-model
+// NOTE: ensure you grant read-only access to any other catalog / schema
+//       you would like the applications to access.
+GRANT USE_CATALOG ON CATALOG <catalog_with_snowplow_data> TO <your_principal_or_user>;
+GRANT USE_SCHEMA ON CATALOG <catalog_with_snowplow_data> TO <your_principal_or_user>;
+GRANT SELECT ON CATALOG <catalog_with_snowplow_data> TO <your_principal_or_user>;
+```
+
+Once created share the following details back with us:
+
+1. [Warehouse server hostname](https://docs.databricks.com/en/integrations/compute-details.html)
+2. [The HTTP path to your SQL Warehouse](https://docs.databricks.com/en/integrations/compute-details.html)
+3. Connection catalog (likely `hive_metastore`)
+4. The default connection schema (indicator of where the app should default into but is configurable in most apps once launched)
+5. [One of](https://docs.databricks.com/en/administration-guide/users-groups/service-principals.html#manage-tokens-for-a-service-principal) (**please send this in a secure format**):
+    1. An OAUTH secret and client ID pair (Coming Soon)
```

Comment:
Should we not have this until it is ready for customers to use?

Reply:
It's down for next sprint so I was hoping we would have it soonish, and give it's databricks preferred approach I was thinking it might be good to highlight to customer that we _will_ offer it given the better security around it?

Reply:
OK that makes sense lets leave it then for now!